docs/pages/views.rb

```diff
@@ -130,6 +130,34 @@ def template(&content)
 					end
 					```
 				MD
+
+				render Markdown.new(<<~MD)
+					## Registering custom elements
+
+					Here's how you can register custom elements.
+				MD
+
+				render Example.new do |e|
+					e.tab "hello.rb", <<~RUBY
+            class Hello < Phlex::View
+              CUSTOM_ELEMENTS = {
+                hello: "hello"
+              }.freeze
+
+              CUSTOM_ELEMENTS.each do |method_name, tag|
+                register_element(method_name, tag)
+              end
```

Comment:
I think the golden path should be to just do this:

```suggestion
              register_element :element_name
```

Reply:
Indeed, that's way cleaner.